src/agent/src/sandbox.rs

```diff
@@ -851,4 +852,241 @@ mod tests {
         let p = s.find_container_process("not-exist-cid", "");
         assert!(p.is_err(), "Expecting Error, Got {:?}", p);
     }
+
+    #[tokio::test]
+    async fn test_find_process() {
+        let logger = slog::Logger::root(slog::Discard, o!());
+        let mut s = Sandbox::new(&logger).unwrap();
+        let (mut linux_container, _root) = create_linuxcontainer();
+
+        // add process with pid of 99
+        let test_pid = 99;
```

Comment:
`libc::pid_t` is actually unsigned, so assuming black-box testing it might be good to test some boundary values too such as `std::i32::MIN`, `-1`, `0`, `1`, `std::i32::MAX` in case there are any conversions happening in `find_process()` or the functions it calls.